### PR TITLE
Crawler with threading

### PR DESCRIPTION
**Pull request by henryyang42**
Dirty

### CODE REVIEW

**Comment by hydai** [comment]
+1

File: crawler/crawler.py [hunk 1]
```diff
@@ -59,11 +59,12 @@ def trim_syllabus(ACIXSTORE, soup):
...
-        a['href'] = a['href'].replace(href_garbage, '').replace(' ', '%20')
+        href = a.get('href', '').replace(href_garbage, '').replace(' ', '%20')
         # Make relative path to absolute path
-        if 'http' not in a['href']:
-            a['href'] = host + a['href']
-
+        if 'http' not in href:
+            href = host + href
+        a['href'] = href
+        a['target'] = '_blank'
```
Comment (hydai) [inline crawler/crawler.py:67]: So we create a new page when user click a link in syllabus?

Comment (henryyang42) [inline crawler/crawler.py:67]: Yes, it is.

**Comment by hydai** [comment]
+1 @henryyang42 Merge it.

**Pull request merged**